pageComponents/NewPostPage/GameTypeSelect.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import Select, { Option } from "@@/components/Select";
+
+interface Props extends React.Attributes {
+  defaultValue?: string;
+  onChange?: (e: React.ChangeEvent<HTMLSelectElement>, value: string) => void;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function GameTypeSelect({
+  defaultValue,
+  onChange,
+  ...props
+}: Props) {
+  return (
+    <Select defaultValue={defaultValue} onChange={onChange} {...props}>
+      <Option value={GameType.cash}>{GameType.cash}</Option>
+      <Option value={GameType.tournament}>{GameType.tournament}</Option>
```

Comment:
Please don't use enum values as UI labels. enum values are just for debugging.

Reply:
Should "Cash" and "Tournament" in my opinion, instead of "CASH" and "TOURNAMENT".